packages/engine/src/gltf/GLTFLoaderFunctions.ts

```diff
@@ -327,9 +330,17 @@ const useLoadBufferView = (options: GLTFParserOptions, bufferViewIndex?: number)
 
 const useLoadBuffer = (options: GLTFParserOptions, bufferIndex) => {
   const json = options.document
-  const result = useHookstate<ArrayBuffer | null>(null)
+  const loader = useHookstate(() => {
+    const fileLoader = new FileLoader(options.manager)
+    fileLoader.setResponseType('arraybuffer')
+    if (options.crossOrigin === 'use-credentials') {
```

Comment:
i think we can eliminate crossOrigin here altogether, we never set it to anything other than `anonymous`